build.assets/Makefile

```diff
@@ -328,3 +277,28 @@ release-arm64-fips: buildbox-arm-fips
 	@if [ -z ${VERSION} ]; then echo "VERSION is not set"; exit 1; fi
 	docker run $(DOCKERFLAGS) $(BCCFLAGS) -i $(NOROOT) $(BUILDBOX_ARM_FIPS) \
 		/usr/bin/make -C e release -e ADDFLAGS="$(ADDFLAGS)" OS=$(OS) ARCH=$(ARCH) RUNTIME=$(RUNTIME) FIPS=yes VERSION=$(VERSION) GITTAG=v$(VERSION)
+
+#
+# Run docs tester to detect problems.
+#
+.PHONY: docsbox
+docsbox:
+	docker pull $(DOCSBOX)
+
+.PHONY: test-docs
```

Comment:
if this is testing `https://github.com/gravitational/next`, should the test logic live there and should the Drone pipeline run on PRs to that repo?

Reply:
I've modified this a bit now so that the tests use a Docker image which is built when https://github.com/gravitational/next pushes to `main`. This means that Teleport doesn't have to be responsible at all for the contents of the image, just mounting the correct docs into the image and running the `yarn` command.